api/src/test/java/org/pmiops/workbench/api/ConceptsControllerTest.java

```diff
@@ -213,6 +240,14 @@ public void testSearchConceptsConceptIdMatch() throws Exception {
             new SearchConceptsRequest().query("456")), CLIENT_CONCEPT_2);
   }
 
+  @Test
+  public void testSearchConceptsMatchOrder() throws Exception{
+    saveConcepts();
+    assertResults(
+            conceptsController.searchConcepts("ns", "name",
+                    new SearchConceptsRequest().query("conceptD")), CLIENT_CONCEPT_4 , CLIENT_CONCEPT_6);
```

Comment:
I'm confused about how this currently passes... CLIENT_CONCEPT_6 is non-standard. Doesn't your code below only return a single concept in this case?

Either way, please add a test that asserts that standard concepts are returned with the non-standard one.

Reply:
I will add one.
The search matches on the code and fetches CLIENT_CONCEPT_4 and matches on the name an d fetches CLIENT_CONCEPT_5 and 6 irrespective of the standard concept filter because the query does not specify the filter.